test/check-application

```diff
@@ -1567,13 +1567,16 @@ class TestApplication(testlib.MachineCase):
         b.click('#run-image-dialog-env-2-btn-close')
         b.click('.env-form .btn-add')
         # Test inputting an key=var entry
-        b.set_val('#run-image-dialog-env-2-value', "RHUBARB=STRAWBERRY DURIAN=LEMON")
+        # testlib.sit()
```

Comment:
Is this (and the other two) something you forgot to remove?

Reply:
Woops, I guess that's what you get when you want to quickly push before going for dinner ;-)